docs/advanced/naming.md

```diff
@@ -1,190 +1,245 @@
-- For `WEB/WEB-DL/WEBRip`, you must include source tags. `AMZN` for Amazon, `DSNP` for Disney+, and `NF` for Netflix.
-- Must contain resolution, i.e, `1080`, `720`, `576` or `480` together with scan type, i.e, `p` if the content is fully Progressive or `i` if it’s Interlaced. So finally it looks like this: `1080p`, `720p`, `576p`, `480i`. Different aspect ratios do not change this resolution, e.g. `1440x1080` and `1920x800` are both still 1080p.
-- Must mention the Video and Audio Codec. There are cases where you have two audio tracks with two different codecs, in which case you must only put the codec of the original/primary/default audio track in the filename.
-- Always add a group tag to your release, ideally placing it at the end for both easier parsing and human readability.
-- You should use `.` or `-` as a delimiter.
-- Limit the use of special characters or characters that need to be escaped. Ideally, don't use any special characters and limit yourself to `a-z`,`A-Z`, and `0-9`. E.g, write `MASH` instead of `M*A*S*H`.
+## File Names
 
-### Filename
+In general, a good file name should include:
 
-- Must contain Season and Episode information in the format `SXXEYY` following [TVDB](https://thetvdb.com/).
-- Absolute Episode Numbers may be used together with `SXXEYY`.
-- Special Episodes must follow the format `S00EXX`.
-- `Group` tag can be at the start or the end of the filename.
+- The full **anime show/series** or **movie title**
+  - If there are multiple versions of the series, you must include the year within the title (e.g. [Hunter x Hunter (1999)](https://thetvdb.com/series/hunter-x-hunter) vs. [Hunter x Hunter (2011)](https://thetvdb.com/series/hunter-x-hunter-2011)). *Note that you must include the year for all movies*
+  - We recommend using names found in databases such as [TheTVDB](https://www.thetvdb.com) (TV series) and [The Movie Database (TMDB)](https://www.themoviedb.org/?language=en-US) (movies)
+- The **season/episode number** (for TV series)
+  - For specials, use `S00EXX` for guaranteed automatic parsing and correct identification. *However, it is suggested to add an additional identifier such as including `OVA` or the episode title (i.e. `S00E01 - OVA Title`) to easily differentiate between episodes*
+- The **source type** (i.e. `BD`, `DVD`, `WEB`)
+  - Append `Remux` if your file uses a Blu-ray/DVD remux (i.e. `BD Remux` or `DVD Remux`)
+  - For `WEB` releases, you may also include the source tag (e.g. `AMZN` for Amazon, `CR` for Crunchyroll, `DSNP` for Disney+, `NF` for Netflix)
+- The **video resolution** and **scan type**
+  - Use `p` for progressive video (e.g. `1080p`, `720p`)
+  - Use `i` for interlaced video (e.g. `1080i`, `720i`)
+- The **video and audio codec**
+  - If your file includes multiple audio tracks in different codecs, only list the default track's codec
+- The **releaser/group tag**
```

Comment:
I don't like merging general and filename section. General was and should be details that cover both filenames and folders while filename should cover only filename